src/protocol/logpoint.ts

```diff
@@ -300,7 +313,7 @@ function eventLogpointMapper(getFrameworkListeners: boolean) {
 
 export async function setEventLogpoint(logGroupId: string, eventTypes: string[]) {
   const mapper = eventLogpointMapper(/* getFrameworkListeners */ true);
-  const analysisId = await createLogpointAnalysis(logGroupId, mapper, null);
+  const analysisId = await createLogpointAnalysis(logGroupId, mapper, null, true);
```

Comment:
should condition be null? probably, but we need to update the type..

Reply:
did you mean the logpoint parameter?